packages/@lwc/babel-plugin-component/src/decorators/index.js

```diff
@@ -22,6 +23,10 @@ const track = require('./track');
 const DECORATOR_TRANSFORMS = [api, wire, track];
 const AVAILABLE_DECORATORS = DECORATOR_TRANSFORMS.map((transform) => transform.name).join(', ');
 
+// FIXME: we're using JavaScript, not TypeScript, and APIFeature is a `const enum`, so we have to hard-code this
+// with the value in `@lwc/shared`'s `api-version.ts`
+const AVOID_DECORATORS_FOR_NON_LIGHTNING_ELEMENT_CLASSES = 3;
```

Comment:
I'm not super happy about this, but I don't see a solution other than converting `@babel/plugin-component` to TypeScript. We have to use `const enum` in `@lwc/shared` because otherwise the `verify-treeshakeable` test fails.

Reply:
This will be fixed by #3373